main.go

```diff
@@ -171,7 +171,7 @@ func main() {
 			SOCKSProxy: "socks5://127.0.0.1:1080",
 			FQDN:       "nitro.nymity.ch",
 			Port:       8080,
-			Debug:      false,
+			Debug:      true,
```

Comment:
Any reason why we need this to be `true`? I prefer it to be `false` in the main branch because the framework's debug mode adds a lot of overhead.

Reply:
It doesn't need to be on by default. I've opened #7 to revert.